book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
```

Comment:
- [ ] A Book has many rentals hence we would require a method for adding rentals in this class. Also, note that the book itself should be sent as a parameter to create a new rental achieving the "has-many" association. I think this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) would be of help to you

Reply:
Well resolved ✔️